azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/implementation/AvailabilitySetImpl.java

```diff
@@ -21,7 +21,7 @@
 import java.util.List;
 
 /**
- * The implementation for {@link AvailabilitySet} and its create and update interfaces.
+ * The implementation for {@link AvailabilitySet} and it's create and update interfaces.
```

Comment:
actually "its" is correct in this form, since its possessive, not a shorthand for "it is"